<?xml version="1.0"?>
<launch>
	<node name="vk_motion_controller" pkg="vk_motion_controller" type="motion_controller" output="screen">
		<rosparam file="$(find vk_motion_controller)/config/parameters.yaml" command="load"/>
	</node>
</launch>